docs/requirements.txt

```
@@ -2,18 +2,20 @@ breathe==4.35.0
 colcon-common-extensions==0.2.1
 colcon-mixin==0.2.0
 doc8==0.10.1
-docutils==0.16.0
+docutils==0.20.1
 GitPython==3.1.41
 setuptools==67.6.0
-sphinx_rtd_theme==0.5.2
-sphinx-tabs==3.2.0
-sphinx==4.3.1
-sphinxcontrib-applehelp==1.0.4
-sphinxcontrib-devhelp==1.0.2
-sphinxcontrib-htmlhelp==2.0.1
+sphinx_rtd_theme==2.0.0
+sphinx-tabs==3.4.5
+# TODO(anyone) remove if https://github.com/sphinx-doc/sphinx/pull/12298/files is merged
+sphinx==7.2.6
```

Following the thread, this has been fixed in sphinx 7.3.6
Should we update to that version and test it? @rsanchez15 

```suggestion
sphinx==7.3.6
```